### PR TITLE
Move cli's Connection export and Variable export command print logic to a seprate function

### PR DESCRIPTION
**Pull request by utkarsharma2**
related: https://github.com/apache/airflow/pull/34640

This PR attempts to move the output logic for `airflow connection export` and `airflow variable export` to a separate function to ensure they don't unknowingly change in the future.

### CODE REVIEW

**Comment by utkarsharma2** [comment]
cc: @hussein-awala

**Comment by jedcunningham** [comment]
@utkarsharma2, it lives here now: https://github.com/apache/airflow/blob/fb92ff8486f21b61a840ddc4414429c3a9adfc88/airflow/auth/managers/fab/cli_commands/user_command.py#L153

File: airflow/cli/utils.py [hunk 1]
```diff
@@ -40,3 +40,10 @@ def is_stdout(fileio: IOBase) -> bool:
     .. warning:: *fileio* must be open for this check to be successful.
     """
     return fileio.fileno() == sys.stdout.fileno()
+
+
+def print_export_output(command_type: str, exported_items: Collection, file_is_stdout: bool, file_path: str):
```
Comment (uranusjr) [inline airflow/cli/utils.py]: It would be better to simply pass in the file object as an argument and calculate both values here.

Comment (utkarsharma2) [reply airflow/cli/utils.py]: @uranusjr Updated the code to reflect the suggestions, please check.

**Comment by utkarsharma2** [comment]
@uranusjr The failure on the PR don't look related.

**Comment by eladkal** [comment]
> @uranusjr The failure on the PR don't look related.

please rebase and lets see how it looks now

**Comment by utkarsharma2** [comment]
@eladkal All checks are passing now.

**Review by uranusjr — APPROVED** [review]

**Pull request merged**